blimflix/client/app/directives/movie-details/movie-details.scss

```diff
@@ -0,0 +1,61 @@
+.modal-content {
+    margin-top: 150px;
+    background-color: #222;
+}
+
+.modal-header {
+    padding-top: 5px;
+    padding-bottom: 5px;
+    border-bottom-color: gray;
+}
+
+.modal-title {
+    color: white;
+}
+
+.modal-body {
+    border-bottom-color: gray;
+}
+
+.modal-footer {
+    border-top-color: gray;
+}
+
+.poster {
+    background-color: white;
+    margin-left: 15px;
+    height: 280px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin-right: 15px;
+}
+
+.info div {
```

Comment:
this changes can affect all over the application. please make it nested
now you are setting global style changes for elements